packages/flutter_tools/test/general.shard/macos/xcode_test.dart

```diff
@@ -788,6 +788,35 @@ void main() {
       "recoverySuggestion" : "Xcode will continue when iPhone is finished.",
       "domain" : "com.apple.platform.iphoneos"
     }
+  },
+  {
+    "modelCode" : "iPad8,5",
+    "simulator" : false,
+    "modelName" : "iPad Pro (12.9-inch) (3rd generation)",
+    "error" : {
+      "code" : -13,
+      "failureReason" : "",
+      "underlyingErrors" : [
+        {
+          "code" : 4,
+          "failureReason" : "",
+          "description" : "iPad is locked.",
```

Comment:
Is this really what the JSON looks like? it says both that it's locked and not connected? that's very confusing...

Reply:
Yes that's exactly copied from my results.  It's not connected, it claims it's a usb interface, but it somehow knows it's locked. But it doesn't actually know, when I unlock the device it doesn't try to wireless pair or something.